crds/operators.coreos.com_clusterserviceversions.yaml

```diff
@@ -4519,6 +4519,19 @@ spec:
                       type: string
                     url:
                       type: string
+                relatedImages:
```

Comment:
my only hesitation to adding these fields is that they are unused on cluster at runtime.

on the other hand, the impact is low -- the change is backwards compatible -- and there are already a few other fields of the spec that can be considered non-runtime metadata (e.g. `provider`, `maturity`, etc).

I'd like to hear what some other folks think since this is still a client facing change.

Reply:
I think it is fine to add those fields. We are using those fields implicitly on the registry side. Perhaps, it helps to make this part more transparent on the API standpoint and easier for folks to use them directly. Plus, as you said it, the change is backwards compatible so there is no risk.

Reply:
From a transparency point of view, would it make sense to add something like the following to the documentation?

```
This field is only used during catalog creation and plays no part in cluster runtime.
```

Also, do we want to add other metadata fields like `provider`, `maturity`? If so, I'd need a list of what those are. I've pieced together documentation from various sources for  `skips` & `relatedImages`, so I can probably do the same for other metadata fields, assuming I know what ones to add. Also the statement I suggested above would apply to these as well.

Reply:
I don't think we want to keep adding more fields to the CSV given that they should really probably end up as properties on bundles and the CSV will go away in the future. If these are necessary for some process temporarily, I think that's fine, but we have been intentionally been avoiding adding any new fields to this type for that reason.

Reply:
I wouldn't say these are new fields. These are existing fields that authors include the CSV but we simply never capture them in CSV API because they don't serve other purposes in runtime. I agree that we would like to avoid adding anything new but for this specific case, I think it is acceptable to do given the "existing" nature of the fields.

Reply:
OK. So I am hearing that this PR should only focus on `skips` & `relatedImages` since we need them for the e2e tests we are writing, and I should add `This field is only used during catalog creation and plays no part in cluster runtime.` to the documentation `skips` & `relatedImages`.

Reply:
Comment updated and force pushed changes

Reply:
Is there anything else that needs to be done for this PR?